hyperbahn_link_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+if [ -z "$1" ]; then
+    dest_dir=$(mktemp -d)
```

Comment:
maybe change this to something like `mktemp -d /tmp/tchannel.XXXXXXX` so it's osx compat